src/bidiServer/index.ts

```diff
@@ -89,30 +91,64 @@ function parseArguments() {
  */
 async function onNewBidiConnectionOpen(
   headless: boolean,
-  chromeChannel: ChromeReleaseChannel,
+  chromeChannel: 'dev' | 'canary' | 'beta' | 'stable',
   bidiTransport: ITransport
 ) {
-  const browserLaunchOptions: PuppeteerLaunchOptions = {
-    headless,
-    channel: chromeChannel,
-  };
+  // See https://github.com/GoogleChrome/chrome-launcher/blob/main/docs/chrome-flags-for-tools.md
+  const chromeArguments = [
+    '--allow-pre-commit-input',
+    '--disable-background-networking',
+    '--disable-background-timer-throttling',
+    '--disable-backgrounding-occluded-windows',
+    '--disable-breakpad',
+    '--disable-client-side-phishing-detection',
+    '--disable-component-extensions-with-background-pages',
+    '--disable-component-update',
+    '--disable-default-apps',
+    '--disable-dev-shm-usage',
+    '--disable-extensions',
+    // AcceptCHFrame disabled because of crbug.com/1348106.
+    '--disable-features=Translate,BackForwardCache,AcceptCHFrame,MediaRouter,OptimizationHints',
+    '--disable-hang-monitor',
+    '--disable-ipc-flooding-protection',
+    '--disable-popup-blocking',
+    '--disable-prompt-on-repost',
+    '--disable-renderer-backgrounding',
+    '--disable-sync',
+    '--enable-automation',
+    '--enable-features=NetworkServiceInProcess2',
+    '--export-tagged-pdf',
+    '--force-color-profile=srgb',
+    '--metrics-recording-only',
+    '--no-first-run',
+    '--password-store=basic',
+    '--use-mock-keychain',
+    '--remote-debugging-port=9222',
+  ];
+  if (headless) {
+    chromeArguments.push('--headless', '--hide-scrollbars', '--mute-audio');
+  }
 
   // 1. Launch Chromium (using Puppeteer for now).
   // Puppeteer should have downloaded Chromium during the installation.
   // Use Puppeteer's logic of launching browser as well.
-  const browser = await puppeteer.launch(browserLaunchOptions);
+  const browser = launch({
+    executablePath: computeSystemExecutablePath({
+      browser: Browser.CHROMIUM,
+      channel: chromeChannel as any,
```

Comment:
Can we do better than `any` here? At the very least, `as string`. But we should have the correct type from puppeteer's `.d.ts` file.